### PR TITLE
echo, fdroid, fdroidcl, ffprobe, ffsend, fg, firefox, fortune, fuck, g++, gdb, gpg2, grep: add German translation

### PR DESCRIPTION
**Pull request by marchersimon**
Translations for:
- echo
- fdroid
- fdroidcl
- ffprobe
- ffsend
- fg
- firefox
- fortune
- fuck
- g++
- gdb
- gpg2
- grep

### CODE REVIEW

File: pages.de/common/gpg2.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+- Liste alle importierten Schlüssel auf:
+
+`gpg2 --list-keys`
+
+- Verschlüssle eine bestimme Datei für einen bestimmten Empfänger und schreibe den Output in eine neue `.gpg` Datei:
+
+`gpg2 --encrypt --recipient {{hans@beispiel.de}} {{pfad/zu/datei.txt}}`
+
+- Verschlüssle eine bestimmte Datei nur mit einem Passwort und schreibe den Output in eine neue `.gpg` Datei
```
Comment (bl-ue) [inline pages.de/common/gpg2.md:15]:
```suggestion
- Verschlüssle eine bestimmte Datei nur mit einem Passwort und schreibe den Output in eine neue `.gpg` Datei:
```

File: pages.de/common/grep.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Suche nach einem exakten Ausdruck:
+
+`grep -F {{exakter_ausdruck}} {{pfad/zu/datei}}`
+
+- Suche nach Ausdrücken [R]ekursiv im aktuellen Verzeichnis, zeige zugehörige Zeilen[n]ummern und [I]gnoriere Binärdateien:
+
+`grep -RIn {{ausdruck}} .`
+
+- Benutze erweiterte reguläre Ausdrücke (unterstützt `?`, `+`, `{}`, `()` und `|`) ohne Beachtung der Groß-, Kleinschreibung:
```
Comment (bl-ue) [inline pages.de/common/grep.md:18]: I don't know German, but this `Beachtung der Groß-,` looks strange.  Is it definitely correct?

**Review by bl-ue — CHANGES_REQUESTED** [review]
Looks good, found a nit and have a question 👍🏻

File: pages.de/common/grep.md (hunk 2 above)
Comment (marchersimon) [reply pages.de/common/grep.md:18]: That's correct

Comment (bl-ue) [reply pages.de/common/grep.md:18]: Interesting, okay, thank you!

**Review by bl-ue — APPROVED** [review]
**13** new ❌ → ✅ on the [i18n page](https://lukwebsforge.github.io/tldri18n) 😋

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**